sats-dna/src/main/kotlin/com/sats/dna/components/SatsTabRow.kt

```diff
@@ -0,0 +1,93 @@
+package com.sats.dna.components
+
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.material3.LocalContentColor
+import androidx.compose.material3.Tab
+import androidx.compose.material3.TabRow
+import androidx.compose.material3.TabRowDefaults
+import androidx.compose.material3.TabRowDefaults.tabIndicatorOffset
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import com.sats.dna.LocalUseMaterial3
+import com.sats.dna.theme.SatsTheme
+
+@Composable
+fun SatsTabRow(
+    selectedTabIndex: Int,
+    modifier: Modifier = Modifier,
+    content: @Composable () -> Unit,
+) {
+    CompositionLocalProvider(LocalUseMaterial3 provides true) {
+        TabRow(
+            selectedTabIndex = selectedTabIndex,
+            containerColor = Color.Transparent,
```

Comment:
Not really sure about these colours yet as this component isn't in the figma library. But I've asked in the Design slack what colours this component should have 🙏

Reply:
Waiting for this before tweaking colors: https://github.com/sats-group/sats-dna-android/pull/437

Reply:
This is available now. 👍